database1/seed.js

```diff
@@ -0,0 +1,32 @@
+const faker = require('faker');
+
+const furnitureArr = [];
+
+//current S3 library holds 101 photos. Will adapt variable when I have more photos.
+const photoLibrarySize = 101;
+//At 1 million there is about 1 second delay to execute function. Plan to seed db in increments of 1m to start --> run createProductEntry to generate 1m, seed db, run createProductEntry again to reach 10m
+const createProductEntry = () => {
+  var photoIndex = 1
+  for (var i = 1; i < 1000001; i ++) {
+    let sampleFurniture = {
+      id: i,
```

Comment:
You actually may want to update this somehow. If you are running this seed 10 times to generate 10,000,000 results, you will actually have 10 items with every single id between 1-1,000,000, since the id restarts at 1 every time you run this function. I'm not positive how different databases would handle this -- a SQL database where id is the primary record wouldn't like this at all! A noSQL database like Mongo probably wouldn't break, because id is just a manually generated property, but you'd have 10 records with each id, meaning your endpoint would either access 10 items at the same time or you wouldn't be able to access 9/10 of your database entries! 
For a SQL database, you could remove the id and let the database auto-increment the id, like I know MySQL allows, but I'm not sure what you could do in a noSQL setting. Something to think about!

Reply:
good point! yea, not planning on using id for sql although I might move forward with neo4j and will def need to think this through more thoroughly if I want to go ahead with the chunked out plan. I think I see what you mean -- like 6 one half dozen another as far as efficiency, and also creating some unintended side effects.